docs/content/how-to-spawn-a-child-workflow-execution-in-go.md

```diff
@@ -0,0 +1,99 @@
+---
+id: how-to-spawn-a-child-workflow-execution-in-go
+title: How to spawn a Child Workflow Execution in Go
+description: Use the `ExecuteChildWorkflow`, available from the `go.temporal.io/sdk/workflow` package, to spawn a Child Workflow Execution in Go.
+tags:
+  - developer-guide
+  - go
+---
+
+import RelatedReadList from '../components/RelatedReadList.js'
+
+Use the [`ExecuteChildWorkflow`](https://pkg.go.dev/go.temporal.io/sdk/workflow#ExecuteChildWorkflow) API, available from the `go.temporal.io/sdk/workflow` package, to spawn a Child Workflow Execution in Go.
+
+The `ExecuteChildWorkflow` call requires an instance of [`workflow.Context`](https://pkg.go.dev/go.temporal.io/sdk@v1.8.0/workflow#Context), with an instance of [`workflow.ChildWorkflowOptions`](https://pkg.go.dev/go.temporal.io/sdk/workflow#ChildWorkflowOptions) applied to it, the Workflow Type, and any parameters that should be passed to the Child Workflow Execution.
+
+`workflow.ChildWorkflowOptions` contain the same fields as `client.StartWorkflowOptions`.
+Workflow Option fields automatically inherit their values from the Parent Workflow Options if they are not explicitly set.
+If a custom `WorkflowID` is not set then one will be generated when the Child Workflow Execution is spawned.
+
+The `ExecuteChildWorkflow` call returns an instance of a [`ChildWorkflowFuture`](https://pkg.go.dev/go.temporal.io/sdk/workflow#ChildWorkflowFuture).
+
+Call the `.Get()` method on the instance of `ChildWorkflowFuture` to wait for the result.
+
+```go
+func YourWorkflowDefinition(ctx workflow.Context, params ParentParams) (ParentResp, error) {
+
+  childWorkflowOptions := workflow.ChildWorkflowOptions{}
+  ctx = workflow.WithChildOptions(ctx, childWorkflowOptions)
+
+  var result ChildResp
+  err := workflow.ExecuteChildWorkflow(ctx, YourOtherWorkflowDefinition, ChildParams{}).Get(ctx, &result)
+  if err != nil {
+    // ...
+  }
+  // ...
+  return resp, nil
+}
+```
+
+**Parent Close Policy**
+
+In Go, a Parent Close Policy is set on the `ParentClosePolicy` field of an instance of [`workflow.ChildWorkflowOptions`](https://pkg.go.dev/go.temporal.io/sdk/workflow#ChildWorkflowOptions).
+The possible values can be obtained from the [`go.temporal.io/api/enums/v1`](https://pkg.go.dev/go.temporal.io/api/enums/v1#ParentClosePolicy) package.
+
+- `PARENT_CLOSE_POLICY_ABANDON`
+- `PARENT_CLOSE_POLICY_TERMINATE`
+- `PARENT_CLOSE_POLICY_REQUEST_CANCEL`
+
+The Child Workflow Options are then applied to the the instance of `workflow.Context`, using the `WithChildOptions` API, which is then passed to the `ExecuteChildWorkflow()` call.
+
+See the **Asynchronous execution** section below for an example.
+
+<RelatedReadList
+readlist={[
+["What is a Parent Close Policy?","/docs/content/what-is-a-parent-close-policy","explanation"],  
+]}
+/>
+
+**Asynchronous execution**
+
```

Comment:
can we use real headers instead of just bolding?

Reply:
no not in /content/ pages
Possible that these could be broken into separate Q&A. But in the end this is a complete answer to the question.